web/common.h

```diff
@@ -0,0 +1 @@
+#define MSG_NOSIGNAL 0 
```

Comment:
@cnlohr Excuse me for this, but didn't know what `MSG_NOSIGNAL` was coming from... what would be a sensible value for that define?